images.go

```diff
@@ -167,6 +179,31 @@ func (r *imageStore) datapath(id, key string) string {
 	return filepath.Join(r.datadir(id), makeBigDataBaseName(key))
 }
 
+// bigDataNameIsManifest determines if a big data item with the specified name
+// is considered to be representative of the image, in that its digest can be
+// said to also be the image's digest.  Currently, if its name is, or begins
+// with, "manifest", we say that it is.
+func bigDataNameIsManifest(name string) bool {
+	return name == ImageDigestBigDataKey || strings.HasPrefix(name, ImageDigestBigDataKey)
```

Comment:
(`HasPrefix` includes the `name == ImageDigestBigDataKey` case. OTOH handling it separately doesn’t hurt at all.)

Reply:
Doh.  Yeah, no need to keep that.